pkg/admin/prerun/prerun.go

```diff
@@ -86,6 +86,20 @@ func (dm *dataManagement) backup() error {
 	}
 	klog.InfoS("Contents of version file", "contents", versionFile)
 
+	currentBootID, err := getCurrentBootID()
```

Comment:
If we're no longer doing a backup on every startup, do we still need the boot ID in the backup name, or could we just use the ostree commit ID?

Other than that question, this looks good.

Reply:
I guess we could get rid of it. Backup's boot ID should be in version file, so it's not like we'd lose any info.
We'd have to change backing up: `create new, delete old` -> `create tmp, rename into place`.
Though I don't think we necessarily need to, there's no penalty for having this, right?

Reply:
If it complicates the backup procedure, I would not touch it.